pkg/metrics/collector.go

```diff
@@ -318,39 +402,45 @@ func (c *ResourcesMetricsCollector) collectRbacAssessmentReports(ctx context.Con
 		for _, r := range reports.Items {
 			labelValues[0] = r.Namespace
 			labelValues[1] = r.Name
-			c.populateRbacAssessmentValues(labelValues, rbacAssessmentDesc, r.Report.Summary, metrics, 2)
+			for i, label := range c.GetReportResourceLabels() {
+				labelValues[i+3] = r.Labels[label]
```

Comment:
I see that in resources that do not have the label , an empty value in shown in metrics , wdyt ? 
`trivy_role_rbacassessments{k8s_label_owner="",name="role-868458b9d6",namespace="kube-system",severity="Critical"} 1`